.github/workflows/main.yml

```diff
@@ -7,17 +7,13 @@ jobs:
       matrix:
         ember-version:
           - 'ember-default'
-          - 'ember-lts-2.16'
-          - 'ember-lts-2.18'
-          - 'ember-3.1'
-          - 'ember-lts-3.4'
-          - 'ember-lts-3.8'
           - 'ember-lts-3.12'
```

Comment:
3.10 is the minimum for classes and decorators and 3.6 is the minimum for classes.  We could install something to get us back to 3.8 but I'm not sure that is necessary if we release a new major bump